Task_2_2_1/src/main/java/r/nsu/fit/masud/Main.java

```diff
@@ -0,0 +1,70 @@
+package r.nsu.fit.masud;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        // Read JSON input file
+        JsonParser parser = new JsonParser();
+        try (FileReader reader = new FileReader("C:\\Users\\Asus\\IdeaProjects\\oop\\Task_2_2_1\\MeinePizza.json")) {
```

Comment:
What if I want to download and run your code? Will it work?

https://www.geeksforgeeks.org/loading-resources-from-classpath-in-java-with-example/

Reply:
yes, it will. I ran it sevral time before PR.